.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior/error:
```

Comment:
Suggest list and code example?
~~~suggestion
Steps to reproduce the behavior/error:
1. 

Code example:
```python
# Your code here
```
~~~